third_party/libpg_query/grammar/statements/select.y

```diff
@@ -2290,7 +2290,23 @@ c_expr:		columnref								{ $$ = $1; }
 			  }
 		;
 
-func_application: func_name '(' ')'
+func_application:
+                        FUNCTION func_name '(' ')'
+				{
+				  	PGFuncCall *n = makeFuncCall($2, NIL, @2);
+					n->is_select_macro = true;
+					$$ = (PGNode *)n;
+					
+				}
+			| FUNCTION  func_name '(' func_arg_list opt_sort_clause opt_ignore_nulls ')'
```

Comment:
I am confused why this is here. Should these macros not only be useable as a table function? Why would the calling syntax need to be changed?